config/initializers/secure_headers.rb

```diff
@@ -16,9 +16,10 @@
       :report_only => false,
       :default_src => ["'self'"],
       :frame_src   => ["'self'"],
-      :font_src    => ["'self'", 'https://fonts.gstatic.com'],
-      :connect_src => ["'self'"],
-      :style_src   => ["'unsafe-inline'", "'self'"],
+      :font_src    => ["'self'", 'https://fonts.gstatic.com',"https://fonts.googleapis.com"],
+      :img_src   => ["'self'", "data:"],
+      :connect_src => ["'self'","'http'","'https:'","'ws:'"],
+      :style_src   => ["'unsafe-inline'", "'self'","https://fonts.googleapis.com","https://fonts.gstatic.com"],
```

Comment:
For future reference, could you add the google fonts that we need this for?
These styles are added dynamically, so it is difficult to grep and find this information.

Reply:
will do it in comments.